frontend/src/services/backend/club_backend.ts

```diff
@@ -0,0 +1,66 @@
+import { ClubInterface } from "../backend_service_interface/backend_service_interface"
+
+/**
+ * Directly connects with the servlet backend to perform club operations.
+ */
+export class ClubBackendService {
+  /** Creates a club on the backend given the club object. */
+  async createClub(club: ClubInterface): Promise<ClubInterface> {
+    const response = await fetch("/api/create-club", {
+      method: "POST",
+      body: JSON.stringify(club),
+    });
+    return response.json();
+  }
+
+  /** Retrieves a list of clubs with some relation to a user. */
+  async listClubs(
+      token: string,
+      membership: "member" | "not member"): Promise<ClubInterface[]> {
```

Comment:
nit: Consider making this an enum instead. No major difference, enums just work better with things like IDE autocomplete and are easier to remember

Reply:
Thank you for this advice! Implemented and pushed.